_data/students.yml

```diff
@@ -73,4 +73,9 @@
   
 - name: Srihari Unnikrishnan
   github: pythongiant
-  image: sr.png
\ No newline at end of file
+  image: sr.png
+  
+- name: Shreesh Kumar Jha
+  github: shreeshjha
+  image: Ai2.png
```

Comment:
rename your image to your github username like `shreeshjha`.

Reply:
I've created the config file and renamed my img as shreeshjha as well